pkg/objectupdate/rte/rte.go

```diff
@@ -148,7 +148,9 @@ func DaemonSetArgs(ds *appsv1.DaemonSet, conf nropv1.NodeGroupConfig) error {
 
 	notifEnabled := isNotifyFileEnabled(&conf)
 	klog.V(2).InfoS("DaemonSet update: event notification", "daemonset", ds.Name, "enabled", notifEnabled)
-	if !notifEnabled {
+	if notifEnabled {
```

Comment:
this is just bug prevention or did you find an actual bug here?

Reply:
The source of the RTE pods spec is derived from the ds args which is in turn derived from the the NROP.  Unless I missed it, the option was not set in this place. It is set by default but in case of any modification and there is no way back to enable it.

Reply:
sure, I can handle that

Reply:
thanks a lot. Let's start checking we have controller tests to exercise this flow. e2e tests here looks like overkill. If we have test coverage and we do the right thing, then it turns into a code readability issue, which is important but less urgent.

Reply:
sure, I'll  revisit and let you know